include/rcpputils/filesystem_helper.hpp

```diff
@@ -183,16 +183,31 @@ inline bool create_directories(const path & p)
 {
   path p_built;
 
```

Comment:
nit: delete this new line and add one between `status` and the `for loop`?

Reply:
A bit of a taste nit, but sure, either way is fine. See https://github.com/ros2/rcpputils/pull/30/commits/6ac7378a787c17871598150f66de498b9c68131a.